### ISSUE TITLE
Enabling plantuml preprocessor seem to conflict with mathjax?

### ISSUE THREAD

**Issue by fulara** [sytsereitsma/mdbook-plantuml#15]
After i enable  plantuml by:  
```
[preprocessor.plantuml]
plantuml-cmd="java -jar /home/fulara/for_others/plantuml.jar"
```
some snippets from mdbook mathjax no longer works specifically: https://github.com/rust-lang/mdBook/blob/master/book-example/src/format/mathjax.md
example:
```
\\( \int x dx = \frac{x^2}{2} + C \\)
```

No longer renders when plantuml is enabled.
but when using begin/end cases it still works.

This probably seems like my configuration issue but I am not 100% sure.

**Comment by sytsereitsma** [comment]
Strange. The preprocessor should only touch/replace the plantuml markdown blocks, it does not inject any js stuff and does not do client side rendering. Could be a bug in the library I use to parse the markdown.
Will have a look

**Comment by sytsereitsma** [comment]
Fixed with v0.6.0-beta
